src/components/pages/auth/login/index.tsx

```diff
@@ -176,6 +178,24 @@ export default function LoginComponent() {
               </button>
             </div>
           </form>
+          <div>
```

Comment:
UI is not in sync with the existing one, please update

Reply:
okay I will do that.

Reply:
I have placed the login with google button inside the form.
Is it ok, or something else should be done here